pkg/kubelet/dockertools/manager.go

```diff
@@ -1552,7 +1552,7 @@ func (dm *DockerManager) runContainerInPod(pod *api.Pod, container *api.Containe
 	containerLogFile := containerInfo.LogPath
 	symlinkFile := LogSymlink(dm.containerLogsDir, kubecontainer.GetPodFullName(pod), container.Name, id.ID)
 	if err = dm.os.Symlink(containerLogFile, symlinkFile); err != nil {
-		glog.Errorf("Failed to create symbolic link to the log file of pod %q container %q: %v", format.Pod(pod), container.Name, err)
+		glog.Errorf("failed to create symbolic link to the log file of pod %q container %q: %v", format.Pod(pod), container.Name, err)
```

Comment:
Error's are addressed [here](https://github.com/golang/go/wiki/CodeReviewComments#error-strings), but not log messages. I believe the convention is for log messages to start with a capital letter.
```console
$ ag --noheading --nobreak 'glog\.[A-Za-z]+\("[A-Z]' pkg | wc -l
869
$ ag --noheading --nobreak 'glog\.[A-Za-z]+\("[a-z]' pkg | wc -l
231
```